src/unix/stream.c

```diff
@@ -1037,17 +1028,11 @@ static int uv__stream_recv_cmsg(uv_stream_t* stream, struct msghdr* msg) {
 }
 
 
-#ifdef __clang__
-# pragma clang diagnostic push
-# pragma clang diagnostic ignored "-Wgnu-folding-constant"
-# pragma clang diagnostic ignored "-Wvla-extension"
-#endif
-
 static void uv__read(uv_stream_t* stream) {
   uv_buf_t buf;
   ssize_t nread;
   struct msghdr msg;
-  char cmsg_space[CMSG_SPACE(UV__CMSG_FD_SIZE)];
+  char cmsg_space[256];
```

Comment:
This seems probably too large (https://github.com/libuv/libuv/pull/2591)

Reply:
I've updated it to reuse the union that we're already using elsewhere, PTAL.